pymobiledevice3/cli/developer.py

```diff
@@ -271,6 +277,32 @@ def screenshot(service_provider: LockdownClient, out):
         out.write(Screenshot(dvt).get_screenshot())
 
 
+def app_lookup(install_service: InstallationProxyService, bundle_id: str):
+    # install_service.lookup is not working for RSD，use browse instead
+    # will get "ConnectionAbortedError" if use lookup
+    app_infos = [v for v in install_service.browse() if v["CFBundleIdentifier"] == bundle_id]
+    if len(app_infos) == 0:
+        raise RuntimeError(f'No app with bundle id {bundle_id} found')
+    return app_infos[0]
+
+
+@dvt.command('xcuitest', cls=Command)
+@click.option("--bundle-id", required=True, help="Bundle ID of the app to test")
```

Comment:
in general this projects prefers `'` instead of `"` (double-quotes) so please refactor them